src/query/events.ts

```diff
@@ -1,5 +1,16 @@
 import { Event, FragmentableArray } from "../generated/prisma-client";
+import moment from "moment";
 
-export const events = (parent, { data }, ctx): FragmentableArray<Event> => {
-  return ctx.prisma.events({ where: data, orderBy: "start_ASC" });
+export const events = (
+  parent,
+  { data, start },
```

Comment:
There is a `start` field in `Event` as well. This makes the `start` argument quite confusing again.